examples/package.json

```diff
@@ -10,7 +10,7 @@
     "build:metadata": "node ./scripts/metadata.mjs",
     "build:sharing": "node ./scripts/sharing-html.mjs",
     "build:standalone": "node ./scripts/standalone-html.mjs",
-    "build:thumbnails": "npm run build && node ./scripts/thumbnails.mjs",
+    "build:thumbnails": "node ./scripts/thumbnails.mjs",
```

Comment:
We discussed this a bit here: https://github.com/playcanvas/engine/pull/5555#issuecomment-1719108268

Basically it allows for a cold start - but I also agree that it takes quite some time before building thumbnails. @mvaligursky Do you have a opinion on this?

Reply:
Well currently this would only be required when modifying or creating new examples now since thumbnails are included in the repo itself now. 

Yea having the thumbnails resolved using the imports I don't like since it splits how the example names are generated from folder and file names to the names specified in the import `index.js` files. One workaround would be setting file path in the `config` to read after the import is resolved but that seems messy.

Additionally you don't need to add example to some index file when generating a new one you can simply add the file within the specified folder.